tasks/setuptools.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Remove system packages for python utilities
+  sudo: yes
+  apt: name={{ item }} state=absent
+  with_items:
+    - python-setuptools
+    - python3-setuptools
+    - python-setuptools-whl
+    - python-pkg-resources
+    - python3-pkg-resources
+    - python-virtualenv
+    - python-tox
+    - python-chardet-whl
```

Comment:
I will remove this.  It is actually installed with either python-tox or python-virtualenv  (and is not used if you then remove these packages.  However there is a much longer list.  e.g.

$ sudo apt-get install python-virtualenv
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  python-chardet-whl python-colorama python-colorama-whl python-distlib
  python-distlib-whl python-html5lib python-html5lib-whl python-pip
  python-pip-whl python-requests-whl python-setuptools python-setuptools-whl
  python-six-whl python-urllib3-whl python-wheel python3-pkg-resources

There is a apt-get autoremove function that will remove unwanted packages, but I feel that is best left for a user to run.